### ISSUE TITLE
Weather Suggestions won't update

### ISSUE THREAD

**Issue by itssoap** [PranshulGG/WeatherMaster#71]
Hi, 

My locality is currently facing a thunderstorm and the same is being reported by Weather Master v1.6.5 (W). But the suggestions for clothing is still relevant for morning/hot weather in general.

The remarks for wearing a t-shirt and the week being hot do make sense logically, but not at the point of a thunderstorm being an active concern.

Thanks

![SmartSelect_20241014_200810_Weather Master.jpg](https://github.com/user-attachments/assets/1c624a59-eace-4083-ba37-a53a69dea1ca)

![SmartSelect_20241014_200732_Weather Master.jpg](https://github.com/user-attachments/assets/4b0382a2-969b-487a-994f-519f416eb1ec)

**Comment by PranshulGG** [comment]
The clothing suggestions are based only on temperature; they don’t check the weather conditions. If you have suggestions for what should be displayed according to weather conditions, I can consider making those changes.

**Comment by itssoap** [comment]
From the pov of India, 29°c can be considered as relatively cool weather in my opinion, so temperature checks and clothing suggestions can be a bit more localized (?). Also, maybe suggestions can be based on UV index as well (wear full sleevs and sunscreen) and/or the AQI levels (should you even leave the home).

Also, from suggestions perspective, as a daily office commuter, I would love to know at the start of the day about what I can expect from the day later (I can definitely infer this from prediction data but still), it would be nice to know if I need to carry an unbrella or a rain-coat, alongside knowing for a fact that the current weather needs so and so precautions.

tl;dr: suggestions to be about present time as well as of later time of the day

**Comment by PranshulGG** [comment]
I’ve been working on the day summary, which provides a better report by calculating the whole day.

![Screenshot 2024-10-14 210837](https://github.com/user-attachments/assets/785e9c40-ac17-425b-a54e-bcb1a539d7ab)


As for the UV index, tips and additional information are available on the UV page if you click on the widget itself.

I’ll improve the clothing suggestions by calculating based on the entire day instead of just the current temperature.

**Comment by itssoap** [comment]
Sounds about right. That covers my expectations well. Thanks!

**Comment by sarojpanda44** [comment]
@

> I’ve been working on the day summary, which provides a better report by calculating the whole day.
> 
> ![Screenshot 2024-10-14 210837](https://private-user-images.githubusercontent.com/134786144/376307869-785e9c40-ac17-425b-a54e-bcb1a539d7ab.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg5Nzc5MjgsIm5iZiI6MTcyODk3NzYyOCwicGF0aCI6Ii8xMzQ3ODYxNDQvMzc2MzA3ODY5LTc4NWU5YzQwLWFjMTctNDI1Yi1hNTRlLWJjYjFhNTM5ZDdhYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxNVQwNzMzNDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMDA4YWUxOWMxZTgzOWY4MzBkMDcxMmRjMzZkMTc5NGRmMGYyOTcyNDY1MTBlNGMwYjQyNzRiMzBlYWMyZDRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.rwgowo5WQBGwcNei8UJ1jdVHT-mTaStFc6UsIHdIS9Y)
> 
> As for the UV index, tips and additional information are available on the UV page if you click on the widget itself.
> 
> I’ll improve the clothing suggestions by calculating based on the entire day instead of just the current temperature.

@PranshulGG about the last statement I would suggest to keep the clothing suggestions based on current tempratue **and** weather condition, instead of the entire day. That would make more sense.
As in this case 29° might be warm or cold, but the weather condition is thunderstorm, so something like "prefer to stay indoor or cary raincoat" will be more logical. See if it makes sense. Thanks.

**Comment by PranshulGG** [comment]
Yeah, that’s much better.

**Comment by PranshulGG** [comment]
@sarojpanda44 It makes more sense to show clothing suggestions like raincoats before it starts raining—it doesn’t make sense to show them when it's already raining.

**Comment by sarojpanda44** [comment]
It's more about prefer to stay inside or (if necessary) carry raincoat. 
I meant it this way

**Comment by itssoap** [comment]
> @
> 
> > I’ve been working on the day summary, which provides a better report by calculating the whole day.
> > 
> > ![Screenshot 2024-10-14 210837](https://private-user-images.githubusercontent.com/134786144/376307869-785e9c40-ac17-425b-a54e-bcb1a539d7ab.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg5Nzc5MjgsIm5iZiI6MTcyODk3NzYyOCwicGF0aCI6Ii8xMzQ3ODYxNDQvMzc2MzA3ODY5LTc4NWU5YzQwLWFjMTctNDI1Yi1hNTRlLWJjYjFhNTM5ZDdhYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxNVQwNzMzNDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMDA4YWUxOWMxZTgzOWY4MzBkMDcxMmRjMzZkMTc5NGRmMGYyOTcyNDY1MTBlNGMwYjQyNzRiMzBlYWMyZDRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.rwgowo5WQBGwcNei8UJ1jdVHT-mTaStFc6UsIHdIS9Y)
> > 
> > As for the UV index, tips and additional information are available on the UV page if you click on the widget itself.
> > 
> > I’ll improve the clothing suggestions by calculating based on the entire day instead of just the current temperature.
> 
> @PranshulGG about the last statement I would suggest to keep the clothing suggestions based on current tempratue **and** weather condition, instead of the entire day. That would make more sense.
> As in this case 29° might be warm or cold, but the weather condition is thunderstorm, so something like "prefer to stay indoor or carry raincoat" will be more logical. See if it makes sense. Thanks.

Suggestions based on current weather is fine, but knowing what to expect over the whole day is necessary as well. App shows weather prediction of the whole day, which is very convenient who wants to plan the whole day or commute mostly, as if its gonna rain, and I have to leave home by 9am, I don't wanna leave without an umbrella and get stuck, or lets say it starts raining at 9:30am, and I m now stuck with no umbrella and somewhere between home and office

**Comment by PranshulGG** [comment]
@itssoap Yeah, I’ve added that to the day summary like I showed you before. Is that fine instead of adding it to the clothing recommendations?

**Comment by itssoap** [comment]
Yep, makes sense for me, I like your approach. A basic overview of the day with necessary cautions would be ideal in my opinion. Thanks
